src/app.ts

```diff
@@ -51,6 +52,7 @@ function prepareRequestContext(req: express.Request): RequestContext {
 export function generateServerlessRouter(
     fhirConfig: FhirConfig,
     supportedGenericResources: string[],
+    headers?: any,
```

Comment:
Avoid using `any`. maybe use a more specific param name.

Also, use an optional options object as the last param so that it's easier to add more optional params in the future. 
Similar to: 
https://github.com/awslabs/fhir-works-on-aws-search-es/blob/30290fdbe15fac047b0b0a76bd4fea7e60f3d62a/src/elasticSearchService.ts#L84-L96